demo/README.link022.md

```diff
@@ -0,0 +1,101 @@
+# Gasket & [Link022](https://github.com/google/link022)
+
+## Getting Started
+First *read* over [Gasket's readme](https://github.com/bairdo/gasket/blob.master/docs/README.authentication.md) for a general overview of Gasket's authentication, and the [link022 readme](./Readme.md).
```

Comment:
Seems this (https://github.com/bairdo/gasket/blob.master/docs/README.authentication.md) links to a NOT FOUND PAGE.

This (./Readme.md) also does not work.